### PR TITLE
Add zapier-platform-core 3.1

### PR DESCRIPTION
**Pull request by bradleyayers**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

File: types/zapier-platform-core/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
+// Type definitions for zapier-platform-core 3.1
+// Project: https://github.com/zapier/zapier-platform-core
+// Definitions by: Bradley Ayers <https://github.com/bradleyayers>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
```
Comment (plantain-00) [inline types/zapier-platform-core/index.d.ts]: It can be Typescript 2.5 here.

**Review by plantain-00 — APPROVED** [review]

File: types/zapier-platform-core/index.d.ts (hunk 1 above)
Comment (bradleyayers) [reply types/zapier-platform-core/index.d.ts]: Does 2.4 imply it *doesn't* work with 2.5? I thought this field was the "oldest" compatible version of TypeScript?

Comment (plantain-00) [reply types/zapier-platform-core/index.d.ts]: Maybe I'm wrong, is there any document about this? I didn't find any.

Comment (bradleyayers) [reply types/zapier-platform-core/index.d.ts]: Yes, see https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-use-features-from-typescript-21-or-above

**Pull request merged**